test/Microsoft.AspNet.Mvc.FunctionalTests/TestHelper.cs

```diff
@@ -47,7 +47,8 @@ public static TestServer CreateServer(Action<IApplicationBuilder> builder, strin
         {
             return TestServer.Create(
                 builder,
-                services => AddTestServices(services, applicationWebSiteName, applicationPath, configureServices));
+                configureServices: null,
+                configureHostServices: services => AddTestServices(services, applicationWebSiteName, applicationPath, configureServices));
```

Comment:
What do you mean?

AddTestServices(
services, 
applicationWebSiteName, applicationPath, configureServices)